js/core/dom_component.js

```diff
@@ -404,19 +404,20 @@ const DOMComponent = Component.inherit({
             .join(' ');
     },
 
-    endUpdate() {
-        const requireRender = !this._initializing && !this._initialized;
+    _updateDOMComponent(requireRender) {
+        if(requireRender) {
+            this._renderComponent();
+        } else if(this._requireRefresh) {
+            this._requireRefresh = false;
+            this._refresh();
+        }
+    },
 
-        this.callBase.apply(this, arguments);
+    endUpdate() {
+        const requireRender = this._isInitializingRequired();
 
-        if(!this._updateLockCount) {
-            if(requireRender) {
-                this._renderComponent();
-            } else if(this._requireRefresh) {
-                this._requireRefresh = false;
-                this._refresh();
-            }
-        }
+        this.callBase();
+        this._isUpdateAllowed() && this._updateDOMComponent(requireRender);
```

Comment:
```suggestion
        this._isUpdateAllowed() && this._updateDOMComponent(this._isInitializingRequired());
```

Reply:
We can't use `this._isInitializingRequired()` here.
After `this.callBase();` we get other values of `this._initializing` and `this._initialized` and other value of `this._isInitializingRequired()`.